doc/tutorials/faq.md

```diff
@@ -0,0 +1,332 @@
+---
+title: Frequently Asked Questions (FAQ)
+layout: doc.hbs
+---
+
+# Frequently Asked Questions
+
+Certain questions arise more often than others at the various places users can
+ask for help. This document tries to list some of the common questions from
```

Comment:
Since we only promote stack overflow now, I would get rid of the first sentence altogether